lib/core-integration/src/Test/Integration/Framework/TestData.hs

```diff
@@ -270,6 +273,14 @@ wildcardsWalletName :: Text
 wildcardsWalletName = "`~`!@#$%^&*()_+-=<>,./?;':\"\"'{}[]\\|❤️ 💔 💌 💕 💞 \
 \💓 💗 💖 💘 💝 💟 💜 💛 💚 💙0️⃣ 1️⃣ 2️⃣ 3️⃣ 4️⃣ 5️⃣ 6️⃣ 7️⃣ 8️⃣ 9️⃣ 🔟🇺🇸🇷🇺🇸 🇦🇫🇦🇲🇸"
 
+
+--
+-- Stake Pools
+--
+
+stakePoolId1 :: Text
+stakePoolId1 = "4f8d686a02c6e625b5a59cc9e234f32e5d72987012f9c25c9a6b60ddade197d1"
```

Comment:
**[change required]**

Use `listStakePools` in integration scenarios to find a stake pool id.

**Why?**

This increases the area of "things-that-have-to-be-changed" every time we make a change in the genesis.yaml. Avoid hard-coded ids as much as possible, especially when they can be easily retrieved by other means.

Reply:
This was intended as a _not-necessarily_ existing pool id… I could change the id and rename it to `nonExistingPoolId`…

Otherwise, fully agree on the "things-that-have-to-be-changed" argument of course.

Reply:
Ah! If it's a non existing pool id, then, a better name would do!